mappings/net/minecraft/block/FungusBlock.mapping

```diff
@@ -2,8 +2,12 @@ CLASS net/minecraft/unmapped/C_naobsnoy net/minecraft/block/FungusBlock
 	FIELD f_agxfxgwm SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_cyyobiev feature Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_ditsybst GROW_CHANCE D
+	FIELD f_kvrossnb requiredBlock Lnet/minecraft/unmapped/C_mmxmpdoq;
```

Comment:
this name is ok, but maybe something that was more clear it was the required *ground* block would be better?

Reply:
Ah, excellent point